### PR TITLE
Add a bio file for @githubteacher

### PR DESCRIPTION
**Pull request by githubteacher**
This Pull Request adds a bio file for @githubteacher as requested in #8.

@githubstudent I would :heart: if you have some :watch: to put some :eyes: on this and let me know your :thought_balloon:s.

### CODE REVIEW

**Comment by githubteacher** [comment]
This looks good but there are a few things that need to change on specific lines.

File: docs/githubteacher.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# GitHub Teacher Bio
+
+- **Name:** GitHub Teacher
+- **Occupation:** Teacher
+- **Location:** San Francisco
```
Comment (githubteacher) [inline docs/githubteacher.md]: We should be more specific.  There are lots of San Franciscos in the world.

Comment (jpenvose) [inline docs/githubteacher.md]: Noted on San Francisco.

Comment (jpenvose) [inline docs/githubteacher.md]: Code looks good!

**Comment by githubteacher** [comment]
:+1: 
:ship: it!
Looks good to me!
:shipit:

**Pull request merged**